tests/drivers/disk/disk_access/src/main.c

```diff
@@ -46,6 +58,43 @@ static uint32_t disk_sector_size;
 /* + 4 to make sure the second buffer is dword-aligned for NVME */
 static uint8_t scratch_buf[2][SECTOR_COUNT4 * SECTOR_SIZE + 4];
 
+#ifdef CONFIG_DISK_DRIVER_LOOPBACK
+#define BACKING_PATH "/"DISK_NAME_PHYS":"
```

Comment:
You could probably create this name from the `DISK_NAME` definition above with preprocessor concatenation, right? Might be worth looking at to avoid maintaining this name in two locations

Reply:
I don't understand, isn't that exactly what is happening? `DISK_NAME_PHYS` is a preprocessor macro.

Reply:
That is- sorry I missed this, was expecting to see `#` for concatenation, but this should work fine as is